### DISCUSSION TITLE
How do I get rid of breadcrumbs?

### DISCUSSION THREAD

**Discussion by blacksqr** [quarto-dev/quarto-cli#6183] [Q&A]
### Description

I find after upgrading to version 1.4.205 that my book project now has breadcrumb navigation links at the top of each chapter page that is within a `part`.  I don't want them there.

Looking in the change notes, I find: 

```
Add support for `bread-crumbs: true|false` to control whether bread crumbs are displayed. Add support for display of 
breadcrumbs on full width (non-mobile) pages when `bread-crumbs` is true. Default value is true.
```

But there is no mention of which configuration category this new option is supposed to go in, and none of my guesses have worked. I have tried `book`, `book>navbar` and `format>html`

There is no mention of the bread-crumbs option in the guide or the reference.

Please let me know how I can set `bread-crumbs: false` and revert my book project to its previous appearance.

**Comment by dragonstyle** [comment]
This is actually a bug that it isn't working (it should be functional under the `book` key). I've just fixed the issue and started a fresh build which should be available within a few minutes. Thank you for letting us know!

**Reply by dragonstyle** [reply]
Version 1.4.217 has the fix.

https://quarto.org/docs/download/prerelease

**Reply by aborruso** [reply]
@dragonstyle I do not find it in the documentation anything about `bread-crumbs: false`.

How to apply to a single page of a book?

I have tried in this way, but it does not work.

````
---
format:
  html:
    toc: false
    page-layout: custom
bread-crumbs: false
page-navigation: false
---
````

Thank you

**Reply by mcanouil** [reply]
That's because it's in 1.4 (as mentioned, and as described in the link provided) and the documentation is being updated for the release.

**Reply by aborruso** [reply]
Thank you.

I had opened also a dev version of the documentation and I didn't find it. But it was probably incomplete.

Do you know how I can apply it?

**Reply by mcanouil** [reply]
> This is actually a bug that it isn't working (it should be functional under the `book` key). I've just fixed the issue and started a fresh build which should be available within a few minutes. Thank you for letting us know!

@aborruso see the above

What are you calling a "page" in a book.
"Pages" in a book are not independent contrary to a website.

**Reply by aborruso** [reply]
I have read it. It's a message of 6 months ago, and it was supposed to be available in a few minutes.
It doesn't work for me today with the latest release.

What did I not understand?

Thank you

**Reply by mcanouil** [reply]
There are some miss-understanding even from the first message here.
- books does not have breadcrumbs at all.
- `bread-crumbs` is a website option and is mentioned in the changelog under "website".
- `page-navigation` is a website/book option and also listed in the changelog under "website" but not in book (documented in https://quarto.org/docs/books/book-structure.html#page-navigation).

This is bread crumbs
<img width="324" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/b593cba8-c717-4ab7-b0b8-033d86ae08f8">

There are no bread-crumbs at all in books, thus a really confusing thread as how to get rid of something that does not exist (or at least I've never seen it so far in any book).

`page-navigation` on the other hand is by default in books.

https://github.com/quarto-dev/quarto-cli/assets/8896044/b153d70b-3413-4139-965c-606d47dda433

If I am missing something, please as usual, share a minimal reproducible example of your book project in which you want to remove "bread-crumbs".

**Reply by aborruso** [reply]
Hi @mcanouil I'm making this question, because I have breadcrumbs in my book (the [repo](https://github.com/ondata/linee-guida-opendata)): https://ondata.github.io/linee-guida-opendata/capitolo-4.html

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/814dc2fd-ab21-46ea-9ccf-2fd3236d6ea4)

**Reply by mcanouil** [reply]
Thanks for the example.
Then indeed there is something wrong as it does not work for books to disable bread-crumbs on a specific page whilst disabling `page-navigation` works.

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/d4e2aa20-0c3e-411b-92bf-b564cebcde53">

@aborruso Could you make a small book with bread-crumbs and open a new issue regarding this?
For example, start from `quarto create project book no-bread-crumbs-page-book` add only what's needed and share that in your issue.

PS: please for the future, take the habit to share small reproducible example. We don't have the time to make guesses and try to make example of something you already have or nearly.

**Reply by aborruso** [reply]
Hi @mcanouil I did not give any examples, because I was only asking what the syntax was.

Now after this exchange with you, it turns out that the syntax is perhaps right, but there might be a bug.
I, on the other hand, thought I was getting the syntax wrong and was asking for documentation and/or what the syntax error was.

I swear I understand that it is good to create reproducible examples. But here I was asking about syntax.

I will try to create example

Thank you

**Reply by mcanouil** [reply]
No worries, even when asking a question about something particular, making an example always help:
1. you to see if it works or not
2. us to see if it should or not

Additionally, misunderstanding is less likely to occur with an example.

**Reply by aborruso** [reply]
Hi @mcanouil the small example page https://aborruso.quarto.pub/sample/intro.html

The code https://github.com/aborruso/sample/blob/main/intro.qmd

Thank you

**Reply by aborruso** [reply]
I'm opening a new issue

**Reply by aborruso** [reply]
The issue https://github.com/quarto-dev/quarto-cli/issues/8109

**Reply by mcanouil** [reply]
I went ahead since I nearly got one when you shared your big book.
